### ISSUE TITLE
Sharing customized character sheet template with campaign players is NOT working

### ISSUE THREAD

**Issue by bennettm74** [worldanvil/worldanvil-bug-tracker#813]
**World Anvil Username**:

bennettm74

**World / Campaign / RPG Character / Manuscript Title**:
World: Da Cool Shit!
Campaign: Da Cool Shit! - Collision of Worlds
RPG Characters: any and all active RPG characters belonging to my invited players

**Feature**: 
Ability for players to choose the custom character sheet template I created

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
I have Grandmaster account access.
I created and saved and marked as shared a modified duplicate of the FATE Character Sheet which I have named Da Cool Shit Character Sheet.
This new template does NOT show up as an option to my players.  They only see the base FATE system templates.
It is worth noting that as the author/storyteller/world creator/campaign creator/template creator, I can create local characters all day long using the templates I created (they all show up in my available templates dropdown) - it is only my players who don't have the option available to them.

**To Reproduce**
I've included the screenshots showing my side where I have a supposedly shared Character Sheet Template called "Da Cool Shit Character Sheet" which is associated with the FATE RPG System and the screenshots of one my players' experience attempting to create a character sheet from template -- the dropdown of available templates does NOT include the customized one.

**Expected behavior**
I expected having paid for Grandmaster level, I could create custom character sheet templates for my players.

**Screenshots**

![image](https://user-images.githubusercontent.com/69326305/149250260-9172e1bd-b955-4246-89fe-dcb9ad8e24be.png)
![image](https://user-images.githubusercontent.com/69326305/149250384-522b8c34-caac-439e-ba6e-bee853ea7b35.png)
![image](https://user-images.githubusercontent.com/69326305/149250640-8eb24119-8cc0-438a-92e8-8910fb0c7c86.png)
![image](https://user-images.githubusercontent.com/69326305/149250721-19c8c062-318c-40a6-a75f-b118474c452b.png)

**Comment by iamromeo** [comment]
Initial tests cannot reproduce the issue.

**Comment by bennettm74** [comment]
Unfortunately, I can't do anything other than reproduce the issue.

I am willing to:

1) Invite an email you give me to be a player in my campaign.  This should
enable you to experience my players' frustration of not having the sheet
template available.
2) Follow whatever steps are required to add you to my world/campaign as an
author/owner or whatever else you need to be to examine my configurations.

-M

**Comment by iamromeo** [comment]
@bennettm74 do not worry this was an initial check note - I will working on this and I will let you know if there is something you acn do to help!

**Comment by bennettm74** [comment]
I have discovered that my custom templates are available to my players IF they ignore their Character Profile screen and instead go to the Statblocks section on the WorldAnvil left-sidebar menu. From there, they can create an independent statblock using the custom templates I've created THEN they can go back to their Character Profile to associate the separately created character sheet to their Character Profile.  

This is a workable work-around for my players, so we are unblocked at this time.  The feature that is STILL not working for us is the originally reported issue that the custom templates do not appear in the Create New Sheet dropdown from the Character Profile screen as they do from the Statblocks screen.  Not sure why the dropdowns differ between the two screens but they do.

Regards -M
